Makefile

```diff
@@ -28,5 +28,9 @@ uninstall:
 	cd plugins/cwe_checker_type_inference; make uninstall; cd ../..
 	cd plugins/cwe_checker_type_inference_print; make uninstall; cd ../..
 
+documentation:
+	dune build @doc
+	cp -r _build/default/_doc/_html doc/html
+
```

Comment:
remove the _build directory?

Reply:
Will be done by `make clean`. However, `make clean` does not delete the contents of *doc/html* and I am not sure whether it should or not.

Reply:
`make clean` deletes *doc/html* now.